clubs_guides/urls.py

```diff
@@ -1,3 +1,18 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+urlpatterns = [
+    url("^$", ClubsGuidesListView.as_view(), name="clubsguide-list"),
+    url(r"^(?P<pk>[0-9]+)", ClubsGuideView.as_view(), name="clubsguide"),
+    url(
```

Comment:
PS. I use 26 size font (coz I'm blind) so widescreen makes no difference :P
It's something I want to add in soon (or will be in the new learning-api), which is https://www.python.org/dev/peps/pep-0008 standards. It is basically a style guide used very commonly for python projects and we would enable it with a linter like Flake8.